hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -277,6 +277,15 @@ public int getParquetPageSize() {
     return Integer.parseInt(props.getProperty(HoodieStorageConfig.PARQUET_PAGE_SIZE_BYTES));
   }
 
+  public int getLogFileDataBlockMaxSize() {
```

Comment:
may be `long` to make it future safe?

Reply:
The problem is we read contents of a log file data block as follows : byte [] bytes = new bytes[size]. Size has to be int.